playbook/app/pb_kits/playbook/pb_select/_select.scss

```diff
@@ -209,10 +209,11 @@
   }
   .pb_select_kit_wrapper {
     &.error {
-      .pb_select_kit_wrapper {
-        > select:first-child {
-          border-color: $error_dark;
-        }
+      > select {
+        border-color: $error_dark;
+      }
+      > div {
```

Comment:
I think we need the error dark color to come from the Body kit. The body kit has a status of `negative` in the select.tsx. That is being styled in the `Body` kit, in the Body's tsx file. That would affect more areas than this obviously, but the change should be coming from that root level.

Reply:
Actually, it might be as simple as passing dark={dark} to https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_select/_select.tsx#L148

That allows the kit to know that it should use dark when dark is true. 

@yuriyivanenko